index.js

```diff
@@ -378,11 +378,13 @@ function Argv (processArgs, cwd) {
   }
 
   var versionOpt = null
-  self.version = function (ver, opt, msg) {
-    versionOpt = opt || 'version'
-    usage.version(ver)
+  self.version = function (opt, msg, ver) {
+    var version = ver || msg || opt || undefined
```

Comment:
@maxrimue :+1: this is what code review is all about, nothing to feel bad about (mind you, I definitely have a guilt complex about some of the terrible code I've added to yargs :stuck_out_tongue: ).